test/Microsoft.AspNet.Mvc.Core.Test/OptionDescriptors/ModelBinderDescriptorExtensionsTest.cs

```diff
@@ -68,5 +69,22 @@ public void ModelBinderDescriptors_AddsTypesAndInstances()
             Assert.Equal(type2, collection[2].OptionType);
             Assert.Equal(type1, collection[3].OptionType);
         }
+
+        [Fact]
+        public void ModelBinders_RemoveTypesOf_RemovesDescriptorsOfIModelBinder()
+        {
+            // Arrange
+            var modelBinders = new MvcOptions().ModelBinders;
+            modelBinders.Add(new ByteArrayModelBinder());
+            modelBinders.Add(Mock.Of<IModelBinder>());
+            modelBinders.Add(typeof(ByteArrayModelBinder));
+            modelBinders.Add(Mock.Of<IModelBinder>());
+
+            // Act
+            modelBinders.RemoveTypesOf<ByteArrayModelBinder>();
+
+            // Assert
+            Assert.False(modelBinders.Any(f => f.OptionType == typeof(ByteArrayModelBinder)));
```

Comment:
nits:
- message when `Assert.False()` fails is basically useless so we tend to avoid it.  better to use `Assert.DoesNotContain(modelBinders, descriptor => descriptor.OptionType == typeof(ByteArrayModelBinder));`
- we also avoid abbreviations even for lambda parameters and even in test code